frontend/device/devicelistener.lua

```diff
@@ -207,6 +207,19 @@ if Device:hasGSensor() then
         return true
     end
 
+    function DeviceListener:onTempGSensorOn()
+        if not G_reader_settings:isTrue("input_ignore_gsensor") then
+            Notification:notify("Accelerometer rotation events already on.")
+        else
+            Device:toggleGSensor()
+            Notification:notify("Accelerometer rotation events on for 5 seconds.")
+            UIManager:scheduleIn(5.0, function()
+                Device:toggleGSensor()
+                end)
```

Comment:
nit: one indent level too deep

Reply:
It's because it's under the `if Device:hasGSensor()` branch. Or the scheduled event should not be indented?